bin/servicemon.py

```diff
@@ -27,27 +27,36 @@
 import threading
 import signal
 import argparse
-from nav import buildconf
+import logging
 
+from nav import buildconf
 import nav.daemon
 from nav.daemon import safesleep as sleep
-from nav.statemon import RunQueue, config, db, debug
+from nav.logs import init_generic_logging, convert_debug_level_to_loglevel
+from nav.statemon import RunQueue, config, db
+
+
+LOGGER = logging.getLogger('bin.servicemon')
 
 
 class Controller:
     def __init__(self, **kwargs):
         signal.signal(signal.SIGHUP, self.signalhandler)
         signal.signal(signal.SIGTERM, self.signalhandler)
         self.conf = config.serviceconf()
-        debug.setDebugLevel(int(self.conf.get('debuglevel', 4)))
+        debuglevel = int(self.conf.get('debuglevel', 4))
+        init_generic_logging(
+            stderr=True, read_config=True,
+            stderr_level=convert_debug_level_to_loglevel(debuglevel)
+        )
         self._deamon = kwargs.get("fork", 1)
         self._isrunning = 1
         self._checkers = []
         self._looptime = int(self.conf.get("checkinterval", 60))
-        debug.debug("Setting checkinterval=%i" % self._looptime)
+        LOGGER.warning("Setting checkinterval=%i", self._looptime)
```

Comment:
The default level to use for a message is hardcoded to **5** in [debug.py](https://github.com/UNINETT/nav/pull/1490/files/58feb4eb1655207462d9dbd855c5375a2567ad20#diff-eaef588d102dc81beafb44a3f7bce47aL50). This does not change that fact. This only changes the filter that decides which messages to actually emit to the log.

It's the difference between

```
logger = logging.getLogger('somelogger')
logger.setLevel(logging.INFO)
```

and 

```
logger = logging.getLogger('somelogger')
logger.info('blah')
```